workshops/07-TPP-Workshop/README.md

```diff
@@ -0,0 +1,82 @@
+# Transformation Priority Premise
```

Comment:
This might be a scary title - maybe it's better to sell the sizzle rather than the sausage?

What about `How to transform your code in steps to end up at awesome`, or `How to build code through tests, step-by-step`?

I definitely think you should mention TPP by name as part of the workshop, but I think it's more exciting if I know, from the headline, what skill I'm aiming at.

Reply:
I can see this for non-academy